website/docs/components/form/textarea/partials/accessibility/accessibility.md

```diff
@@ -1,8 +1,22 @@
-#### Known Issues
+## Conformance rating
+
+### Form::Textarea::Field
+
+<Doc::Badge @type="success">Conformant</Doc::Badge>
+
+`Form::Textarea::Field` is conformant when lorem ipsum
+
+### Form::Textarea::Base
+
+<Doc::Badge @type="warning">Conditionally conformant</Doc::Badge>
```

Comment:
@MelSumner do you know what the conformance of this is (or where I should look to know this on my own)?